### ISSUE TITLE
Merge clips with embedded srt?

### ISSUE THREAD

**Issue by raj6996** [Zulko/moviepy#1361]
I have 50 very short video clips (~1-10 sec duration) with 50 separated .srt files. Already merged with ffmpeg, but cant setup properly embeded srt file in merged video.
Want to join/merge clips with soft coded srt. How to make it?
I don't type clip1 clip2 etc. 
want any batch script to merge using moviepy.

**Comment by tburrows13** [comment]
You could do something like this:

```
filenames = ["file1", "file2", "file3", ...]
clips = []
for filename in filenames:
    clip = VideoFileClip(filename)
    # Do stuff in here with subtitles?
    clips.append(clip)

final_clip = concatenate_videoclips(clips)
```

**Comment by raj6996** [comment]
> $ python concent.py
> Traceback (most recent call last):
>   File "concent.py", line 6, in <module>
>     clip = VideoFileClip(filename)
>   File "C:\Python38\lib\site-packages\moviepy\video\io\VideoFileClip.py", line 88, in __init__
>     self.reader = FFMPEG_VideoReader(filename, pix_fmt=pix_fmt,
>   File "C:\Python38\lib\site-packages\moviepy\video\io\ffmpeg_reader.py", line 35, in __init__
>     infos = ffmpeg_parse_infos(filename, print_infos, check_duration,
>   File "C:\Python38\lib\site-packages\moviepy\video\io\ffmpeg_reader.py", line 270, in ffmpeg_parse_infos
>     raise IOError(("MoviePy error: the file %s could not be found!\n"
> OSError: MoviePy error: the file C_001 could not be found!
> Please check that you entered the correct path.
I used C_001, C002, C003 etc named file upto 50 files, I pasted code inside videoclip folder.

**Comment by tburrows13** [comment]
Well that simply means that the file `"C_001"` does not exist - there's not much I can help you with there. Usually they have an extension, so I'd expect it to be called `"C_001.mp"` or similar.

**Comment by raj6996** [comment]
python concent.py
> Traceback (most recent call last):
>   File "concent.py", line 6, in <module>
>     clip = VideoFileClip(filename)
>   File "C:\Python38\lib\site-packages\moviepy\video\io\VideoFileClip.py", line 88, in __init__
>     self.reader = FFMPEG_VideoReader(filename, pix_fmt=pix_fmt,
>   File "C:\Python38\lib\site-packages\moviepy\video\io\ffmpeg_reader.py", line 35, in __init__
>     infos = ffmpeg_parse_infos(filename, print_infos, check_duration,
>   File "C:\Python38\lib\site-packages\moviepy\video\io\ffmpeg_reader.py", line 244, in ffmpeg_parse_infos
>     is_GIF = filename.endswith('.gif')
> AttributeError: 'ellipsis' object has no attribute 'endswith'
>

**Comment by tburrows13** [comment]
You copied my line `filenames = ["file1", "file2", "file3", ...]`, but the `...` was just an example. Make sure that your list has only strings in it.

**Comment by raj6996** [comment]
```
from moviepy.editor import *

filenames = ["001.mp4", "002.mp4", "003.mp4", "004.mp4"]
clips = []
for filename in filenames:
    clip = VideoFileClip(filename)
    # Do stuff in here with subtitles?
    clips.append(clip)

final_clip = concatenate_videoclips(clips)
```
Nothing happened?

**Comment by tburrows13** [comment]
Sorry, but I'm not just going to write your entire code here. If there's a specific question that you have that you can't answer by googling or reading the documentation then I can help you with it, but I can't just give you the whole thing. For a start, I don't even know exactly what you're trying to do.

**Comment by raj6996** [comment]
![moviepy](https://user-images.githubusercontent.com/16426916/96903193-2b1db200-14b3-11eb-8d85-16ff7b8c200b.jpg)
Just want to merge file with closed captioned srt file.
I just moved files a new folder to test.
That folder has total 50 files with 50 srt file just like image.
I merged with videos with ffmpeg, but cant add srt files being very short clips(1 or 3 sec.).
I googled about it last 4 days, but nothing found proper answer though I have tested many codes. I recall moviepy can do it.
so i asked questions. Hope it is clarify to you.
